src/cromshell/utilities/HTTPUtils.py

```diff
@@ -0,0 +1,27 @@
+import logging
+import sys
+
+import requests
+
+from cromshell.utilities import IOUtils
+
+LOGGER = logging.getLogger(__name__)
+
+
+def assert_can_communicate_with_server(cromshell_url):
+    """Check Connection with Cromwell Server"""
+
+    try:
+        curl_out = requests.get("{}/api/workflows/v1/backends".format(cromshell_url))
```

Comment:
F-strings.  Pull out string into variable - in this case it should reference the same `v1` api end point that has been  defined in another variable (see an earlier comment) and append the `/backends` suffix to it.

Reply:
done